### ISSUE TITLE
Doesn't work locale 'ru' with showMonthYearPicker

### ISSUE THREAD

**Issue by YevhenKysilov** [Hacker0x01/react-datepicker#1699]
I set locale

import DatePicker, { registerLocale } from "react-datepicker";
import ru from "date-fns/locale/ru";

componentWillMount() {
    registerLocale("ru", ru);
  }

and create datepicker
`
<DatePicker
selected={this.state.date}
showMonthYearPicker
dateFormat="YYYY-MM"
locale="ru"
/>
`

Datepicker language is "en"

It didn't work with showMonthYearPicker, but without showMonthYearPicker all is good

**Comment by golubvladimir** [comment]
package.json

`"date-fns": "^2.0.0-alpha.23",`

component.js

```
import ru from 'date-fns/locale/ru';
...
<DatePicker
  locale={ru}
>
```

**Comment by MihailShev** [comment]
package.json
`"date-fns": "2.0.0-alpha.27",`

component.js
Not work: `import ru from 'date-fns/locale/ru';`
Work: `import * as ru from 'date-fns/locale/ru';`

` <DatePicker locale={ru}/>`

**Comment by YevhenKysilov** [comment]
> package.json
> `"date-fns": "2.0.0-alpha.27",`
> 
> component.js
> Not working: `import ru from 'date-fns/locale/ru';`
> Working: `import * as ru from 'date-fns/locale/ru';`
> 
> ` <DatePicker locale={ru}/>`

It doesn't work with showMonthYearPicker

**Comment by bochkarev-artem** [comment]
The same as YevhenKysilov described. Also doesnt work for `de` locale. Whenever option `showMonthYearPicker={true}` is used, translation is not working, showing default english names of monthes.

**Comment by gautam-pahuja** [comment]
Thanks for reporting guys, the fix is not that big, any of you want to give a try and open a PR?

**Comment by bochkarev-artem** [comment]
Actually issue is fixed already, tried updating to 2.8.0 version and it works there. The problem was that function was passing locale as second parameter, but function declaration expected locale as 3rd parameter
@YevhenKysilov could you confirm and close it?

**Comment by gnepud** [comment]
package.json
` "react-datepicker": "^2.9.6",`
```
import fr from 'date-fns/locale/fr';
...
<DatePicker
  locale={fr}
>
```
It works, thanks
